<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vigenere_Window</class>
 <widget class="QMainWindow" name="Vigenere_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>585</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>585</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Vigenere (Made by Borshch Dmytro)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Input_Field">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>581</width>
      <height>201</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Copy your text here </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Your text</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="Output_Field">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>581</width>
      <height>201</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>There will be result</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Hack_Button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hack!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Decrypt_Button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>230</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Encrypt_Button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>230</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Key_Field">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string>Key word</string>
    </property>
    <property name="placeholderText">
     <string>Key word</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Key length</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Key_Length_Monitor">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Accept_Key_Button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_Button">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>520</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear_Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Exit_Button</sender>
   <signal>clicked()</signal>
   <receiver>Vigenere_Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
